VaticanBAV/et/BAVet203.xml

```diff
@@ -0,0 +1,228 @@
+                    </additions>
+                    <bindingDesc>
+                      <binding contemporary="true">
+                        <decoNote xml:id="b1">Two wooden boards.</decoNote>
+                        <decoNote xml:id="b2" type="bindingMaterial">
+                          <material key="wood"/>                           
+                        </decoNote>
+                      </binding>
+                    </bindingDesc>
+                  </physDesc>
+                  
+                  <history>
+                    <origin>
+                      <origDate notBefore="1800" notAfter="1899"/>         
+                      The manuscript belonged to <persName ref="PRS14685WaldaSa" role="owner">Walda Samāʿt</persName>, whose name is mentioned in some supplication 
+                      formulas (e.g., on <locus target="#36v"/>). The name of a possibly later owner, <persName ref="PRS14686WaldaMa" role="owner">Walda Māryām</persName>, 
+                      is mentioned in <ref target="#a1"/>. 
+                    </origin>
+                    <provenance>According to the account on the manuscripts’ provenance given by Eugène Tisserant, this manuscript is among those
+                      acquired by <persName ref="PRS4805Grebaut" role="owner"/> in <date when="1925"/> with funds of
+                      <persName ref="PRS14055SaintJ">Jean Saint</persName>. 
+                      The manuscripts originated from Šawā (ʾAddis ʾAbabā, Dabra Libānos, Dabra Bǝrhān),
+                      Goǧǧām (Dimā Giyorgis, Marṭula Māryām, Dabra Warq) and Bagemdǝr (Gondar, ʾAzazo Takla Hāymānot) and very few from Tǝgrāy.
+                    </provenance>
+                    <acquisition when="1928">The manuscript entered the library on <date>18 April 1928</date>.</acquisition>
+                  </history>
+                  <additional>
+                    <adminInfo>
+                      <recordHist>
+                        <source>
+                          <listBibl type="catalogue">
+                            <bibl>
+                              <ptr target="bm:GrebTiss1935Codices"/>
+                              <citedRange unit="page">673-675</citedRange>
+                            </bibl>
+                          </listBibl>
+                          <listBibl type="secondary">
+                            <bibl>
+                              <ptr target="bm:GrebTiss1936Codices"/>
+                              <citedRange unit="page">20</citedRange>
+                            </bibl>
+                          </listBibl>
+                        </source>
+                      </recordHist>
+                    </adminInfo>
+                  </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+      <encodingDesc>
+        <p>A digital born TEI file</p>
+      </encodingDesc>
+      <profileDesc>
+        <particDesc>
+          <listPerson>
```

Comment:
I just have noticed that you describe relations in a different way. Is it an alternative that does not affect anything?

Reply:
I haven't found references in the guidelines. Perhaps @thea-m knows more on how people involved should be indicated and if the present way is ok?

Reply:
> I haven't found references in the guidelines. Perhaps @thea-m knows more on how people involved should be indicated and if the present way is ok?

she will be away for a while, but we were using `relations` saws:hasOwned [](uhttps://betamasaheft.eu/Guidelines/?q=relations&start=6&id=relationsrl) (if you check Person to Manuscript)t.

Reply:
I think you are right, it is in pers records! I am confusing you and myself! I am sorry!

Reply:
Don't worry Nafisa!